client/settings/dpcs-settingspanel/panel_evoker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ "$BASH_SOURCE" == "$0" ]]
+then 
+	sudo -s python2 settingspanel.py
```

Comment:
The problem was, the graphical application was run, and when the `Apply` or `OK` button were pressed, the error was shown in the console. The user might not see the error and the application doesn't respond.

So, if we are not going to hard-code sudo, the exception has to be caught and properly handled.